hw10.py

```diff
@@ -0,0 +1,54 @@
+# 1. До завдання, в якому ви реалізовували телефонну книгу, додати валідацію номера телефону за допомогою RegEx.
+# Врахувати можливість декількох форматів: +380XXXXXXXXX, 380XXXXXXXXX, 0XXXXXXXXX
+import json
+import re
+enter = input('''You can see all contacts(stats), all names(list), someone\'s number(show) or (add), (delete) contacts.
+Enter your command: ''')
+if enter == 'add':
+    name = input('Enter name to add: ')
+    phone = input('Enter phone to add: ')
+    match1 = re.search('\+380\d{9}', phone)
+    match2 = re.search('380\d{9}', phone)
+    match3 = re.search('0\d{9}', phone)
+    try:
+        if match1:
+            m = match1[0]
```

Comment:
You don't need to use the value from Match object, you can just check if any(match1, match2, match3) and add user-entered phone